3.analyze_data/utils/significance_testing.py

```diff
@@ -44,7 +43,35 @@ def store_comparisons(_comp_functions, _treatments, _dmso_probs, _treatment_prob
 
     return _treatments
 
-def get_treatment_comparison(_comp_functions, _treatment_paths, _probadf):
+def filter_wells_by_cell_count(_df, _cutoff):
+    """
+    Parameters
+    ----------
+    _df: pandas Dataframe
+        The dataframe which contains the well, which will be potentially removed
+
+    _cutoff: Integer
+        The number of cells a well must have to remain in the dataframe
+
+    Returns
+    -------
+    _df: pandas dataframe
```

Comment:
```suggestion
    _df: pandas.DataFrame
```

Reply:
This is to denote that the `DataFrame` object is a part of the `pandas` package. Consider adding but doesn't change overall function

Reply:
Consider applying to the rest of the occurrences as well.

Reply:
Changed! Also did this for pandas series